### ISSUE TITLE
Settings can't set older Kubernetes versions

### ISSUE THREAD

**Issue by mattfarina** [rancher-sandbox/rancher-desktop#321]
In the settings 1.16.7 is the older version that can be set. 1.13.0 is the oldest in the json file containing the versions.

<img width="941" alt="Screen Shot 2021-06-02 at 6 29 33 AM" src="https://user-images.githubusercontent.com/62991/120465531-128c4900-c36c-11eb-8cf1-2dac8f922424.png">

I've reproduced on both mac and windows.

**Comment by jandubois** [comment]
> 1.13.0 is the oldest in the json file containing the versions.

Not sure which file you are referring to (it is probably not used anymore), but we need to download the releases from Github, and `v1.16.7` is the oldest release available:

```console
$ gh api repos/k3s-io/k3s/releases --paginate | jq -r .[].name | sort -V -u | grep -E ^v1.[1-9]
v1.16.7+k3s1
v1.16.7-rc1+k3s1
v1.16.9+k3s1
[...]
v1.21.1+k3s1
v1.21.1-rc1+k3s1
v1.21.1-rc2+k3s1
```

**Comment by jandubois** [comment]
On a somewhat related note: we really should have a test that the old versions actually work on both our macOS and Windows VMs. I only ever use v1.19+ during manual testing.

There may be bugs/hidden assumptions about the environment in earlier releases that have only been fixed in more recent ones.

**Comment by ericpromislow** [comment]
In the code:
```
protected readonly minimumVersion = new semver.SemVer('1.15.0');
…
 if (version < this.minimumVersion) {
      console.log(`Version ${ version } is less than the minimum ${ this.minimumVersion }, skipping.`);

      // We may have new patch versions for really old releases; fetch more.
      return true;
    }
```

And when I hit the API, there are no non-rc released versions between 1.0.1 and 1.16.7. So, for now, the code is working as designed. I assume we aren't interested in supporting versions predating 1.16.
